features/promotion/applying_catalog_promotions/applying_catalog_promotions_price_calculation.feature

```diff
@@ -0,0 +1,47 @@
+@applying_catalog_promotions
+Feature: Applying percentage catalog promotions
+    In order to check prices calculation
+    As a Visitor
+    I want to see discounted products in the catalog
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store classifies its products as "Soft Drinks"
+        And the store has a "Orange Juice" configurable product
+        And this product belongs to "Soft Drinks"
+        And this product has "Orange Juice" variant priced at "$20"
```

Comment:
```php
And the store has an "Orange Juice" product priced at "$20"
And this product belongs to "Soft Drinks"
```

would be simpler and has all the required information 🖖 The same with other products